requirements.txt

```diff
@@ -1,4 +1,9 @@
 nest-asyncio>=1.2.0
 qiskit-terra>=0.10
+qiskit-aer
+qiskit-ibmq-provider
```

Comment:
Why are aer and the ibmq provider required. There shouldn't be a hard dependency on this besides terra.

Reply:
We can fix the warnings by just setting `QISKIT_SUPPRESS_PACKAGING_WARNINGS=Y`in the env

Reply:
Nice.  Added to tox.ini.